.vitepress/theme/custom.css

```
@@ -1,48 +1,192 @@
-/* PhenoDocs Custom Styles */
+/* ═══════════════════════════════════════════════════════════════
+ * Keycap Palette - Phenotype Shared VitePress Theme
+ *
+ * Accent:       #7ebab5 (teal)
+ * Primary text: #f6f5f5 (warm white) / #1a1c1e (dark ink)
+ * Dark BG:      #090a0c (deep midnight)
+ * Light BG:     #f8f9fa (warm paper)
+ * Secondary:    #353a40 (slate)
+ * ═══════════════════════════════════════════════════════════════ */
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap');
```

Stylelint (17.4.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap'" (import-notation)

(import-notation)